content/news/2021-01-24-internet-censorship.md

```diff
@@ -0,0 +1,123 @@
+title: "About my Outreachy internship with Mixxx DJ: Contributing to open source amidst Uganda’s internet restrictions"
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2021-01-24 14:00:00
+summary: Last week, amid the 2021 general election, Uganda was disconnected from the internet entirely. The country experienced a widespread internet blackout that lasted 4 days, starting on the eve of the election (13th January 2021) and ending in the morning of 18th January 2021. In the days leading up to the election, access to major social media platforms and circumvention tools was blocked – even when the OTT (Over the Top) tax (commonly referred to as the “Social Media Tax”) was paid. This internet shutdown and the social media ban that followed affected many online jobs and businesses, my internship with Mixxx under the Outreachy program was no exception.
```

Comment:
this should be the first paragraph of the article, not the summary.  The summary could be something like:

"In the days leading up to the Ugandan election, access to major social media platforms and circumvention tools was blocked. This internet shutdown and the social media ban that followed affected many online jobs and businesses, my internship with Mixxx under the Outreachy program was no exception."

Reply:
Thank you